lib/node_modules/@stdlib/blas/base/zaxpy/README.md

```diff
@@ -171,6 +172,14 @@ var alpha = new Complex128( 2.0, 2.0 );
 // Scale values from `x` by `alpha` and add the result to `y`:
 zaxpy( x.length, alpha, x, 1, y, 1 );
 
+// Print the results:
+logEach( '(%s)*(%s) + (%s) = %s', alpha, x, yc, y );
+
+yc = zcopy( y.length, y, 1, zeros( y.length, 'complex128' ), 1 );
```

Comment:
I think you want to be copying `yc`, not `y`, here, as `yc` is a copy of the original random array. You mutated `y` at L173.